Notice.md

```diff
@@ -6,50 +6,99 @@ the open-source TEE maintained by Linaro, with initial contributions from
 STMicroelectronics, Ericsson and Linaro Limited.
 
 What OP-TEE is
-------
 
-OP-TEE is designed primarily to rely on the ARM TrustZone(R) technology as the
-underlying hardware isolation mechanism. However, it has been structured to be
-compatible with any isolation technology suitable for the TEE concept and goals,
-such as running as a virtual machine or on a dedicated CPU.
+OP-TEE is a Trusted Execution Environment designed as companion to a non secure
+Linux kernel running on ARM&reg; Cortex-A cores using the TrustZone&reg;
+technology. OP-TEE meets the TEE System Architecture and provides the TEE
+Internal Core API v1.1 to Trusted Applications and the TEE Client API
+v1.0, all as defined by the [GlobalPlatform TEE specifications].
+
+The non secure OS is referred to as the Rich Execution Environment (REE) in TEE
+specifications. It is typically a Linux OS flavor as a GNU/Linux OS or an
+Android.
+
+OP-TEE is designed primarily to rely on the ARM TrustZone; technology as
```

Comment:
`;`?

Reply:
old `&reg;` not fully removed. fix. thanks.